src/iter/par_bridge.rs

```diff
@@ -1,12 +1,9 @@
-use crossbeam_deque::{Steal, Stealer, Worker};
-
-use std::sync::atomic::{AtomicBool, AtomicUsize, Ordering};
-use std::sync::{Mutex, TryLockError};
-use std::thread::yield_now;
+use std::sync::Mutex;
 
 use crate::current_num_threads;
 use crate::iter::plumbing::{bridge_unindexed, Folder, UnindexedConsumer, UnindexedProducer};
 use crate::iter::ParallelIterator;
+use std::sync::atomic::{AtomicBool, AtomicUsize, Ordering};
```

Comment:
Can you move this up with the other `use std::` statement? Or move the other one down, either way is fine as long as it's consistent. (I think `rustfmt` doesn't touch the order of "groups" separated by empty lines.)